src/styles/header.scss

```diff
@@ -0,0 +1,73 @@
+$moyo--color: #00acdc;
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
```

Comment:
Please check, header must be fullwidth
![image](https://user-images.githubusercontent.com/54936044/224472877-70c13784-7744-4959-8c16-0f72260c7dbb.png)